### ISSUE TITLE
Shower instant max fill 

### ISSUE THREAD

**Issue by HolyBanana1** [naelstrof/KoboldKare#195]
![Screenshot 2023-03-06 014221-min](https://user-images.githubusercontent.com/126751635/223048363-c663fc46-8afe-4af0-a0f7-bb049602955b.png)
im not sure if its intentional or not. But if you leave the shower running for extended periods of time(30s-5 min)  it seems like it stores the fluid amount and the next object or receptacle placed underneath it will fill to the saved value.  the image is of me placing a heart under the shower after 5 min. it instantly grew to max size. again im not sure if this is intentional or a bug.

**Comment by naelstrof** [comment]
Yeah I've been aware of this bug, I probably should fix it but I didn't want users to feel like they were losing fluids during transfers (so I made it impossible).

I should probably have a time-out or a limit on it.
